Duckling/Quantity/EN/Corpus.hs

```diff
@@ -46,14 +46,16 @@ allExamples = concat
              ]
```

Comment:
Also more use cases for pound, from the ticket: `(pound, lb, us pound)`

Reply:
The pound is ambiguous and cannot be inferred out of context, therefore the system returns two hits for the word `pound`. One as a currency and the other as a quantity unit. I am afraid I cannot make it smarter with the current system in an easy way as the context and the semantics and meaning in the sentence are not understood by Duckling. I can add more test cases. The ones you see there are written by Facebook's team.